### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by MPDADDY**
Hi👏,
I have styled the App to look similar to the the sample provided.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <div className="books_header">
+      <div className="nav">
+        <h3 className="dark_blue">Bookstore</h3>
+        <h4 className="gray_title">Books</h4>
+        <h4 className="fade_title">Categories</h4>
+      </div>
+      <div>profile</div>
+    </div>
+  );
+}
```
Comment (Abenezer-Tilahun) [inline src/components/Header.js]: - [ ] As stated in the project requirement **Your final design should match Zeplin's design (this includes all elements + the same styling)** So Kindly I recommend you to update your header for title 'Bookstore CMS' and  the 'Profile' Just like the given design **Kindly use an Person Icon just like the below Header design** .
- The Zeplin's design Header Look likes this : 

header


- Your Header Design Look like this : 
header

File: src/components/Book.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+.book-progress {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
```
Comment (Abenezer-Tilahun) [inline src/components/Book.css:21]: - [ ]  As stated in the project requirement **Your final design should match Zeplin's design (this includes all elements + the same styling)** So Kindly I recommend you to book progress section to contain 'A circle update status icon' Just like the Given design so Kindly Fix it .

- The Zeplin's design Book progress section Look  likes the below : 
book

- Your Book Progress design Look likes this : 
book

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MPDADDY 

Great job for implemented  the some of the Project Requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git-Flow
✔️ Good commit messages
✔️ Professional README
✔️  Correctly added All elements in Zeplin's design.


almost

## Required Changes ♻️
_Check the comments below the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MPDADDY** [comment]
@Abenezer-Tilahun,  implemented the changes requested.

**Review by Munsa1 — APPROVED** [review]
Hi @MPDADDY, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes  By Previous Reviewer :heavy_check_mark: 


## Optional suggestions
- [ ] In your own time I would encourage you to closely follow the given design. Some requirements were missed like using reusable classes for repeating rules, things such as the font  and font-weight can also improve a given design

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MPDADDY** [comment]
@Munsa1, I appreciate it, bro.

**Pull request merged**